paket.dependencies

```diff
@@ -15,33 +15,33 @@ source https://api.nuget.org/v3/index.json
 // nuget Microsoft.AspNetCore.All 2.0.0
 
 // DOTNETCORE & PAKET ISSUES
-nuget Microsoft.NETCore.App 2.2.0
-nuget Microsoft.NETCore.DotNetAppHost 2.2.0
-nuget Microsoft.NETCore.DotNetHostResolver 2.2.0
-nuget Microsoft.NETCore.DotNetHostPolicy 2.2.0
+nuget Microsoft.NETCore.App 2.2.4
+nuget Microsoft.NETCore.DotNetAppHost 2.2.4
+nuget Microsoft.NETCore.DotNetHostResolver 2.2.4
+nuget Microsoft.NETCore.DotNetHostPolicy 2.2.4
 nuget NETStandard.Library 2.0.3
 
 // PRODUCTION STUFF
 nuget Microsoft.Extensions.Configuration 2.2.0
 nuget Microsoft.Extensions.Configuration.CommandLine 2.2.0
 nuget Microsoft.Extensions.Configuration.FileExtensions 2.2.0
-nuget Microsoft.Extensions.Configuration.EnvironmentVariables 2.2.0
+nuget Microsoft.Extensions.Configuration.EnvironmentVariables 2.2.4
 nuget Microsoft.Extensions.Configuration.Json 2.2.0
 
-nuget Autofac 4.9.1
+nuget Autofac 4.9.2
 nuget Autofac.Extensions.DependencyInjection 4.4.0
 
-nuget NetTopologySuite 1.15.1
+nuget NetTopologySuite 1.15.2
 
 nuget Serilog 2.8.0
```

Comment:
You can remove all Serilog things from paket.dependencies, we've moved them to https://github.com/Informatievlaanderen/api/blob/master/src/Be.Vlaanderen.Basisregisters.Api/paket.template#L92-L99